packages/p2p/src/components/buy-sell/popup.jsx

```diff
@@ -0,0 +1,186 @@
+import React, { Fragment, Component }      from 'react';
+import PropTypes                           from 'prop-types';
+import { Formik, Field, Form }             from 'formik';
+import { Input, Button, ThemedScrollbars } from 'deriv-components';
+import { localize, Localize }              from 'deriv-translations';
+import IconBack                            from '../../assets/icon-back.jsx';
+import IconClose                           from '../../assets/icon-close.jsx';
+
+class Popup extends Component {
+
+    handleSubmit = (formik_vars, { setSubmitting }) => {
+        // TODO: p2p API call to create ad
+        // eslint-disable-next-line no-console
+        console.log(this.state);
+        // eslint-disable-next-line no-console
+        console.log(formik_vars);
+        setSubmitting(false)
+    }
+
+    render() { 
+        const { ad, onCancel } = this.props;
+        const is_buy = ad.type === 'buy';
+        const amount_currency = ad.min_transaction;
+        const amount_asset = ad.min_transaction / ad.fix_price;
+        const initial_send = is_buy ? amount_currency : amount_asset;
```

Comment:
const [initial_receive, send_currency, receive_currency] = getInitialValues(is_buy) ?

Reply:
thats the thing i need...., man, thank you 🙏